cli/command/image/build_test.go

```diff
@@ -18,6 +20,52 @@ import (
 	"golang.org/x/net/context"
 )
 
+func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Invalid test on Windows")
+	}
```

Comment:
We have some helpers for this, please use https://godoc.org/github.com/gotestyourself/gotestyourself/skip

```go
skip.IfCondition(t, runtime.GOOS == "windows", "uid and gid not relevant on windows")
```